x-pack/plugins/case/server/client/cases/push.ts

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
a global remark -> I think we should have multiple try catch and be smart about the error we send back to the user to let them know that something wrong happen and where. It might make our life easier in our SDH life